llvm/test/DebugInfo/KeyInstructions/Generic/simplifycfg-branch-fold.ll

```diff
@@ -0,0 +1,81 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt %s -S -passes=simplifycfg -bonus-inst-threshold=2 | FileCheck %s
+
+;; Block d gets folded into preds b and c. Check the cloned instructions get
```

Comment:
Copy the block diagram from this review into here?

Reply:
I'm not sure where I got the diagram in the description from. I've added one that actually describes this multi-pred test.